src/app/shared/services/account/account.service.ts

```diff
@@ -73,6 +75,12 @@ export class AccountService {
       this.setRootFolder(new FolderVO(cachedRoot));
       this.refreshRoot();
     }
+
+    this.httpv2.tokenExpired.subscribe(() => {
```

Comment:
don't you need to unsubscribe to this? or reloading the window would do the trick

Reply:
Because there is only ever one "global" instance of the AccountService, it doesn't get destroyed until the window is actually reloaded. So we don't need to unsubscribe, although it might be a good idea to do so anyway!